src/g1.rs

```diff
@@ -257,9 +257,57 @@ impl G1Affine {
         }
     }
 
+    /// Raw bytes representation
+    ///
+    /// The intended usage of this function is for trusted sets of data where performance is
+    /// critical. This way, the `infinity` internal attribute will not be stored and `x` and
+    /// `y` will be stored without any check.
+    ///
+    /// For secure serialization, check `to_compressed` and `to_uncompressed`
```

Comment:
Not sure `to_compressed` and `to_uncompressed` exist anymore. Also, `to_bytes` and `from_bytes` should be the referenced fn's.